src/main/java/com/fundacionjala/movies/Movie_Childrens.java

```diff
@@ -0,0 +1,30 @@
+package com.fundacionjala.movies;
+
+/**
+ * Created by danielgonzales on 6/11/2016.
+ */
+public class Movie_Childrens extends Movie {
+
+    private static final double RENT_PRICE = 1.5;
```

Comment:
If you notice the Movie_Childrens, Movie_New_Release, MovieRegular classes, you are repeating the RENT_PRICE , PRICE_RETARD, POINTS variables in all those classes.
Please abstract those variable into the Movie class